hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/WritableECContainerProvider.java

```diff
@@ -218,13 +218,14 @@ private long getConfiguredContainerSize() {
   @ConfigGroup(prefix = "ozone.scm.ec")
   public static class WritableECContainerProviderConfig {
 
-    @Config(key = "pipeline.minimum",
-        defaultValue = "5",
+    @Config(key = "pipeline.limit",
```

Comment:
This parameter was not intended to be the limit, but the minimum number of pipelines we should have so we don't only have one pipeline and always allocate blocks to the same one.

We need to finish off HDDS-5327 and decide on a strategy based on cluster size and load to determine the pipeline limit.

As the code stands at the moment, this parameter indicates the number of pipelines that will end up open at any given time. It is like a minimum and a limit. Once we figure out how to handle the limit, then this parameter will be the minimum number of pipelines. Therefore it probably makes sense to keep the parameter named pipeline.minimum rathern than limit.

Reply:
Thanks for the explanation. I'll change it back.